test/functional/apps/discover/index.ts

```diff
@@ -55,6 +55,9 @@ export default function ({ getService, loadTestFile }: FtrProviderContext) {
     loadTestFile(require.resolve('./_chart_hidden'));
     loadTestFile(require.resolve('./_context_encoded_url_param'));
     loadTestFile(require.resolve('./_data_view_editor'));
+    if (config.get('esTestCluster.ccs')) {
+      loadTestFile(require.resolve('./_data_view_editor'));
+    }
```

Comment:
I'm a bit confused: for CCS it will load file 2 times?

Reply:
If you want to run only few test files, you can explicitly set it in the config file:

```
testFiles: [require.resolve('./apps/discover/_data_view_editor.ts')],
```

Reply:
@LeeDr @cuff-links what do you think?

Reply:
Thanks. Fixed.